EIPS/eip-xxxx.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Multiple Native Tokens
```

Comment:
```suggestion
title: Native Tokens
```
Is multiple superfluous as tokens is plural?

Reply:
Good point!